### ISSUE TITLE
geo_point does not support range queries while select by not null

### ISSUE THREAD

**Issue by wb14123** [crate/crate#6126]
**CrateDB version**:

2.0.6

**JVM version**:

openjdk version "1.8.0_144"                         
OpenJDK Runtime Environment (build 1.8.0_144-b01)   
OpenJDK 64-Bit Server VM (build 25.144-b01, mixed mode) 

**OS version / environment description**:

Archlinux. Kernel 4.12.4.

**Problem description**:

When I select geo_point field filter by not null, it throws error:

```
select * from geo_test where geo is not null;
```

It responses with: 

```
SQLActionException[UnhandledServerException: java.lang.IllegalArgumentException: Field [geo] of type [geo_point] does not support range queries]
```

The table structure is like this:

```
create table geo_test (geo geo_point, id long);
```

**Steps to reproduce**:

Create the table:

```
create table geo_test (geo geo_point, id long);
```

Insert data:

```
insert into geo_test (geo_point, id) values ([1, 1], 1);
insert into geo_test (id) values (2);
```

Query:

```
select * from geo_test where geo is not null
```

**Comment by mfussenegger** [comment]
Thanks for reporting,

we're able to reproduce it and will take a closer look at it.

**Comment by mfussenegger** [comment]
We've merged a fix which will be included in the next `2.1` release.
